README.md

```diff
@@ -101,5 +101,252 @@ With this, we are defining a mapping from namespaces to directories. The `src/`
+
+echo $user->getName();
+?>
+```
+
+This will work. However, to take advantage of PHP namespaces, you can import a class from a namespace instead of importing the namespace. Add the the following line just below `require '../src/Model/User.php';`:
+```
+use SallePW\Model\User;
+```
+
+The `use` operator lets you import the `User` class from the `SallePW\Model` namespace. Therefore, we don’t have to prefix the class name with the namespace.
+```
+$user = new User(1, "Nicole Marie Jimenez"); // you can directly use the User class
+```
+
+What if there are more classes inside the same namespace? Add another class called `Book` inside the `Model` folder.
+```
+<?php
+
+declare(strict_types=1);
+
+namespace SallePW\Model;
+
+final class Book {
+  public function __construct(
+    private string $name, 
+    private string $genre,
+    private string $author,
+    private int $year)
+    {
+    }
+
+    public function getName() {
+      return $this->name;
+    }
+
+    public function getGenre() {
+      return $this->genre;
+    }
+
+    public function getAuthor() {
+      return $this->author;
+    }
+
+    public function getYear() {
+      return $this->year;
+    }
+}
+```
+
+To be able to use both classes, you can either import them individually:
+```
+require '../src/Model/User.php';
+require '../src/Model/Book.php';
+
+use SallePW\Model\User;
+use SallePW\Model\Book;
+
+$user = new User(1, "Nicole Marie Jimenez");
+$book = new Book("Design Systems", "Design", "Alla Kholmatova", 2017);
+
+echo $user->getName();
+echo $book->getName();
+```
+
+There is a way to simplify this. Both of these classes are in the same namespace. You have previously defined in the `composer.json` the namespaces. Instead of having to use `require` for each file individually, you can take advantage of the namespaces. First, remove the following lines:
+```
+require '../src/Model/User.php';
+require '../src/Model/Book.php';
+```
+
+Check the browser again. What error do you see? Why do you think PHP raised that error?
+
+Now, just below the `declare(strict_types=1);` line, add the `autoload.php` file which is inside the `vendor/` folder. This will load the namespaces defined in `composer.json`.
+
+```
+<?php
+
+declare(strict_types=1);
+
+require_once __DIR__ . '/../vendor/autoload.php';
+
+use SallePW\Model\User;
+use SallePW\Model\Book;
+```
+
+With this, you will be able to use the `User` class and `Book` class to instantiate new objects. These two classes are the classes that you have created yourself. Now, it's time to use the classes that come from the packages or dependencies that you installed using **composer**.
+
+# Using packages
+
+First, we want to generate fake data with the [**Faker**](https://github.com/fzaninotto/Faker/blob/master/readme.md) package. According to the documentation, you can start using the package by creating the Faker:
+```
+use Faker\Factory; // using the Factory class included in the Faker package
```

Comment:
Explain why this thisrd-party package is also "found" thanks to the autoload.php